bolt-modules/boltlib/lib/puppet/functions/run_script.rb

```diff
@@ -80,7 +80,7 @@ def run_script_with_description(scope, script, targets, description = nil, optio
         Puppet::Pops::Issues::NOT_A_FILE, file: script
       )
     end
-
+    executor.report_file_source(self.class.name, script)
```

Comment:
In general I chose to report once the file has actually been found (so paths that do not exist, possibly due to user error) will not be reported. Is this the correct behavior?